feature/bgp/policybase/otg_tests/default_policies_test/bgp_default_policies_test.go

```diff
@@ -690,6 +657,7 @@ func testDefaultPolicyAcceptRouteActionOnly(t *testing.T, dut *ondatra.DUTDevice
 
 	t.Run("Create and apply default-policy ACCEPT-ALL with action as ACCEPT_ROUTE", func(t *testing.T) {
 		if deviations.BgpDefaultPolicyUnsupported(dut) {
+			gnmi.Delete(t, dut, gnmi.OC().RoutingPolicy().Config())
 			configureRoutingPolicyDefaultAction(t, dut, "accept", replacePolicy)
```

Comment:
The `replacePolicy` flag in the function was with the intention to achieve the `Delete` operation performed here.

Reply:
fixed by doing a gnmi.Replace() if replace is True